src/ripple/app/ledger/LedgerEntrySet.cpp

```diff
@@ -64,6 +69,7 @@ void LedgerEntrySet::swapWith (LedgerEntrySet& e)
     mSet.swap (e.mSet);
     std::swap (mParams, e.mParams);
     std::swap (mSeq, e.mSeq);
+    boost::swap (mFundingTable, e.mFundingTable);
```

Comment:
Stupid question: Why `boost::swap` instead of `std::swap` like the other members here?